### PR TITLE
Refactor: first item in fusions[] is forge target

### PR DESCRIPTION
**Pull request by RojjaCebolla**
this makes it easier to sort lines alphabetically by the *output* of the recipe instead of by whichever input happens to be input 1.

this also makes it easier for us to glance over the list and see what's currently producible by #forgery and what is not.

### CODE REVIEW

File: src/fountain.c [hunk 1]
```diff
@@ -747,15 +751,15 @@ doforging(void)
...
+            if ((obj1->otyp == fusions[i][1] && obj2->otyp == fusions[i][2]) ||
+                (obj2->otyp == fusions[i][1] && obj1->otyp == fusions[i][2])) {
+                obj1->otyp = fusions[i][0];
                 break;
             }
         }
     }
     /* Take on the secondary object's material. */
-    if (valid_obj_material(obj1, obj2->material)) obj1->material = obj2->material;
+    // if (valid_obj_material(obj1, obj2->material)) obj1->material = obj2->material;
```
Comment (RojjaCebolla) [inline src/fountain.c:762]: whoops, this should be a separate commit. however, removing this seems to be necessary for making wand forging work correctly.

**Review by NullCGT — APPROVED** [review]
